tests/lib/rules/comma-dangle.js

```diff
@@ -68,11 +76,11 @@ ruleTester.run("comma-dangle", rule, {
         { code: "var foo = { bar:\n\n'bar' }", options: [ "only-multiline" ] },
         { code: "var foo = {a: 1, b: 2, c: 3, d: 4}", options: [ "always-multiline" ]},
         { code: "var foo = {a: 1, b: 2, c: 3, d: 4}", options: [ "only-multiline" ]},
-        { code: "var foo = {a: 1, b: 2,\n c: 3, d: 4}", options: [ "always-multiline" ]},
-        { code: "var foo = {a: 1, b: 2,\n c: 3, d: 4}", options: [ "only-multiline" ]},
+        { code: "var foo = {a: 1, b: 2,\n c: 3, d: 4,}", options: [ "always-multiline" ]},
+        { code: "var foo = {a: 1, b: 2,\n c: 3, d: 4,}", options: [ "only-multiline" ]},
         { code: "var foo = {x: {\nfoo: 'bar',\n}}", options: [ "always-multiline" ]},
         { code: "var foo = {x: {\nfoo: 'bar',\n}}", options: [ "only-multiline" ]},
-        { code: "var foo = new Map([\n[key, {\na: 1,\nb: 2,\nc: 3,\n}],\n])", options: [ "always-multiline" ]},
+        { code: "var foo = new Map([\n[key, {\na: 1,\nb: 2,\nc: 3,\n},],\n])", options: [ "always-multiline" ]},
```

Comment:
This one I had to change because now it is considered multiline. Please check this behaves as expected.